keylime/ca_util.py

```diff
@@ -64,6 +64,7 @@
 
 
 def globalcb(*args):
+    del args
```

Comment:
this function is not used, we could simply remove it.

Reply:
Also remove the commented caller?
```

        # code to create a pem formatted protected private key using the keystore
    #     pk = EVP.load_key_string(str(priv[0][name]))
    #     f = BIO.MemoryBuffer()
    #     # globalcb will return the global password provided by the user
    #     pk.save_key_bio(f, 'aes_256_cbc', globalcb)
    #     prot_priv = f.getvalue()
    #     f.close()
```

Reply:
It seems we tend to remove password protection for the private key in the PR #426
but as a crypto newbie, this comment provides some reference probably useful, keeping it at the moment seems fine.
So, I guess we comment it either?

Reply:
Commented now...